src/pages/contact-us/_affiliates.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Text } from '../../components/elements/typography'
+import { Header } from 'components/elements'
+import { localize } from 'components/localization'
+import device from 'themes/device'
+
+const Wrapper = styled.section`
+    width: 100%;
+    height: 19rem;
+    background-color: var(--color-grey-8);
+    padding: 5.6rem 12.8rem;
+    display: flex;
+    flex-wrap: nowrap;
+    flex-direction: row;
+    justify-content: space-around;
+
+    @media ${device.tabletL} {
+        flex-wrap: wrap;
+        height: auto;
+    }
+`
+const Content = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+`
+const StyledHeader = styled(Header)`
+    margin-bottom: 0.8rem;
+`
+const StyledText = styled(Text)`
+    margin-bottom: ${props => props.marginBttom || '0.8rem'};
+    color: ${props => props.color || 'var(--color-black-3)'};
```

Comment:
we have color props on `Text` component as well, and its also default on color-black-3